### ISSUE TITLE
Glyphs with Unicode Point over 0xFFFF Missing in .TTF File

### ISSUE THREAD

**Issue by robloo** [microsoft/fluentui-system-icons#299]
Glyphs with a Unicode Point over 0xFFFF appear to be missing in the .ttf files here:

https://github.com/microsoft/fluentui-system-icons/tree/master/fonts
`FluentSystemIcons-Filled.ttf` and `FluentSystemIcons-Regular.ttf`

Example of these glyphs are:

```json
    "ic_fluent_play_16_regular": "0x10029",
    "ic_fluent_play_28_regular": "0x1002a",
    "ic_fluent_play_32_regular": "0x1002b",
    "ic_fluent_receipt_money_24_regular": "0x1002c",
    "ic_fluent_save_16_regular": "0x1002d",
    "ic_fluent_shield_error_16_regular": "0x1002e",
    "ic_fluent_sport_soccer_16_regular": "0x1002f",
    "ic_fluent_square_20_regular": "0x10030",
```

This is a pretty big issue unless I'm missing something. This was found when trying to switch an app over to the Fluent UI System Icons and some Icons were just not appearing.

**Comment by antranapp** [comment]
Yes, I see the same problem.  `ic_fluent_play_16_regular` is not rendered correctly.

![Simulator Screen Shot - iPhone 12 Pro - 2021-07-31 at 18 39 55](https://user-images.githubusercontent.com/478757/127737317-afb8e1bf-d599-42c4-8581-3e38ad0b8988.png)

@rickromano I am writing an icon browser screen in iOS and seeing this. Do you have any idea?

**Comment by robloo** [comment]
This seems like a pretty big issue but I've seen no movement on it. 

I've also noted that Unicode points that are very high (nearing 0xFFFF) don't actually work on all platforms like Android. Android treats them like they are right-to-left and doesn't properly measure them. I think Microsoft has too many icons and they are just going outside the special use codepage.

**Comment by robloo** [comment]
In order to work-around this issue I just re-built the fonts using my own tooling. I highly suggest Microsoft fixes tooling on their end though.

The corrected fonts:
 * https://github.com/robloo/SymbolIconManager/blob/main/Fonts/FluentUISystemFilled.ttf
 * https://github.com/robloo/SymbolIconManager/blob/main/Fonts/FluentUISystemRegular.ttf

*These are missing ~24 glyphs which could not be automatically found likely due to file name errors in this repository

**Comment by spencer-nelson** [comment]
The tooling is all in the repository here. Please feel welcome to submit PRs to fix the issue.

**Comment by 1nf0rmatix** [comment]
Push
